src/js/plugins/plugin.text_selection.js

```diff
@@ -0,0 +1,179 @@
+//@ts-check
+const BookReader = /** @type {typeof import('../BookReader').default} */(window.BookReader);
+
+class TextSelectionPlugin {
+
+  constructor() {
+    /**@type {PromiseLike<JQuery<HTMLElement>>} */
+    this.djvuPagesPromise = null;
+  }
+
+  /**
+   * @param {string} ocaid
+   */
+  init(ocaid) {
+    this.djvuPagesPromise = $.ajax({
+      type: "GET",
+      url: `https://cors.archive.org/cors/${ocaid}/${ocaid}_djvu.xml`,
+      dataType: "xml",
+
+      error: function (e) {
+        return undefined;
+      }
+    }).then(function (response) {
+      const xmlMap = response;
+
+      if (xmlMap != undefined) {
+        return $(xmlMap).find("OBJECT");
+      }
+    });
+  }
+
+  /**
+   * @param {number} index
+   * @returns {Promise<HTMLElement>}
+   */
+  async getPageText(index) {
+    return (await this.djvuPagesPromise)[index];
+  }
+
+  /**
+   * @param {JQuery} $container
+   */
+  stopPageFlip($container){
+    const $svg = $container.find('svg');
+    $svg.on("mousedown", (event) => {
+      if ($(event.target).is('tspan')) {
+        event.stopPropagation();
+        $container.one("mouseup", (event) => event.stopPropagation());
+      }
+    });
+  }
```

Comment:
It would be great if we could avoid this function; I'm not sure why these event handlers need to get re-attached after zoom/flip; shouldn't they already be there? It did like the SVG was getting reconstructed after zoom, but this fn is called inside `createTextLayer`, so shouldn't it already be attached...? Not sure :P